modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='gs-publishing-walkthroughs-proc']
+= Adding {walkthrough-name}s to your {PRODUCT} cluster
+
+The home page of the web app lists all the walkthoughs from all the git repositories you are subscribed to.
+Any user can add {walkthrough-name}s to your cluster.
+See xref:gs-writing-walkthroughs-proc[] for information about writing {walkthrough-name}s.
+
+This procedure describes how to subscribe your cluster to a Git repository that contains {walkthrough-name}s.
+
+.Procedure
+
+
+. Navigate to the *Solution Explorer* URL in a new browser tab.
+
+. Click the gear icon in the top right to display the *Application settings* screen.
+
+. Enter the URLs of the walkthrough repositories you want to add to your cluster. For example:
++
+----
+https://github.com/integr8ly/example-customisations.git#master
+----
++
+[NOTE]
+====
+* Enter one URL per line.
+* To include a specific branch, use the syntax `#<branch-name>`.
+* List URLs in the order you want them to appear on the Home page.
+====
+
+. Click *Save*.
++
+Editing the value triggers an automatic redeployment of the Webapp.
+
+. When the deployment is complete, refresh the Webapp in your Browser.
++
+You should now see new walkthroughs available from the dashboard.
++
+[NOTE]
+====
+If the git repository is updated with new content, the web app is not automatically updated. You must:
+
+. Open the link:{openshift-host}[Openshift Console].
```

Comment:
Is this link correct?  The link resolves to an "openshift-host" file in the partials directory but can't see any sign of it in this PR.

Reply:
oops, updated